src/screens/Customers/AddPayment.tsx

```diff
@@ -61,15 +64,15 @@ function AddPayment(props: AddPaymentProps) {
 
     const remainingBalance = currentCustomer.totalAmountBilledfromInvoices - currentCustomer.totalAmountPaidfromPayments;
 
-    const cardInfo = [{ number: remainingBalance.toString(), label: 'Remaining Balance', unit: 'Ks' }]
+    const cardInfo = [{ number: remainingBalance.toString(), label: intl(words.remaining_balance), unit: intl(words.ks) }]
 
     return (
-        <BaseScreen title="Add Payment" leftIcon="backNav">
+        <BaseScreen title={intl(words.add_x, words.payment)} leftIcon="backNav">
             <div className={classes.amountOwedContainer}>
                 <OutlinedCardList info={cardInfo} />
             </div>
-            <TextField label={'Amount Paid (Ks)'} id={'amount-paid'} onChange={handleSetPaymentAmountInput} />
-            <Button label={'+ ADD PAYMENT'} onClick={handleSubmit} loading={loading} />
+            <TextField label={intl(words.amount_spent_paid) + ' (' + intl(words.ks) + ')'} id={'amount-paid'} onChange={handleSetPaymentAmountInput} />
```

Comment:
NIT: this might look cleaner as a string template so you don't need to do the parentheses concatenations